container-images/scripts/rag_framework

```diff
@@ -107,7 +125,12 @@ class Rag(cmd.Cmd):
             self.vectordb = qdrant(vector_path)
 
         # Setup openai api
-        self.llm = openai.OpenAI(api_key="your-api-key", base_url="http://localhost:8080")
+        self.port = os.getenv("RAMALAMA_PORT", "8080")
+        url = f"http://localhost:{self.port}"
+
+        wait_for_llama_server(url, timeout=20)
```

Comment:
NIT: The value for `timeout` passed here is the same as the default, so I'd simply remove it here.